app/pages/profile/comment-link.cjsx

```diff
@@ -57,13 +57,13 @@ module.exports = React.createClass
           <span>
             {' '}in{' '}
             <a href={@state.href}>
-              {if @state.project? and @state.owner
+              {if @state.project? and @state.owner? and !@props.project?
```

Comment:
I was just trying to figure that out too.  I suppose this would be true if you were viewing a user profile outside of a project.  e.g. `/users/parrish`.  That would have `@props.project` (the project context) undefined and `@state.project` (the project of the comment) defined.

Definitely confusing though.  Maybe some renaming would help?